codegen/src/getLanguageFromExtension.ts

```diff
@@ -0,0 +1,14 @@
+import type { Language } from './codegen';
+import { errors } from './errors';
+
+export const getLanguageFromExtension = (extension?: 'ts' | 'js') => {
```

Comment:
This function is missing the return type. Otherwise we are not using the `Language` type here in this file, even though it is imported.

```suggestion
export const getLanguageFromExtension = (extension?: 'ts' | 'js'): Language => {
```

Reply:
Also, the `Language` type can be simplified now to:

> export type Language = 'typescript' | 'javascript';

Reply:
Thanks, done in https://github.com/xataio/client-ts/pull/52/commits/e5f4de371cc4a75cfc78bb3a074c6ec7c1785136.